invenio_rdm_records/services/pids/providers/external.py

```diff
@@ -85,6 +85,14 @@ def validate(self, record, identifier=None, provider=None, client=None, **kwargs
 
         return (True, []) if not errors else (False, errors)
 
+    def validate_record(self, record):
```

Comment:
why don't we make this the default implementation and get rid of the `@abstractmethod`? I guess most of the times, there will no be requirement to implement this...what do you think?